Doctrine/Orm/Extension/PaginationExtension.php

```diff
@@ -55,7 +55,7 @@ public function __construct(ManagerRegistry $managerRegistry, RequestStack $requ
     public function applyToCollection(ResourceInterface $resource, QueryBuilder $queryBuilder)
     {
         $request = $this->requestStack->getCurrentRequest();
-        if (null === $request || !$this->isPaginationEnabled($resource, $request)) {
+        if (!$this->isPaginationEnabled($resource, $request)) {
```

Comment:
http://api.symfony.com/2.7/Symfony/Component/HttpFoundation/RequestStack.html#method_getCurrentRequest

> Outside the handling of a request, $requestStack->getCurrentRequest() returns null.
http://symfony.com/blog/new-in-symfony-2-4-the-request-stack